### PR TITLE
[python3] build.py with python3 (mostly bytestring)

### PR DESCRIPTION
**Pull request by lethliel**
* return of get_buildinfo is bytestring with python3
* other variables contain bytestrings as well now

needs https://github.com/openSUSE/osc/pull/464 to work

### CODE REVIEW

File: osc/core.py [hunk 1]
```diff
@@ -6081,6 +6081,7 @@ def get_dependson(apiurl, project, repository, arch, packages=None, reverse=None
     return f.read()
 
 def get_buildinfo(apiurl, prj, package, repository, arch, specfile=None, addlist=None, debug=None):
+    print('In the function')
```
Comment (z3ntu) [inline osc/core.py]: unnecessary print statement

File: osc/build.py [hunk 2]
```diff
@@ -771,13 +772,15 @@ def main(apiurl, opts, argv):
...
             print('Getting buildinfo from server and store to %s' % bi_filename)
-            bi_text = ''.join(get_buildinfo(apiurl,
-                                            prj,
-                                            pac,
-                                            repo,
-                                            arch,
-                                            specfile=build_descr_data,
-                                            addlist=extra_pkgs))
+
+            print('HERE')
```
Comment (z3ntu) [inline osc/build.py]: unnecessary print statement

**Comment by lethliel** [comment]
@z3ntu Thanks for the catch. Removed the debug print statements.

**Comment by lethliel** [comment]
merged as discussed with @marcus-h on IRC

**Pull request merged**